gatsby-site/src/components/Documents.js

```diff
@@ -0,0 +1,98 @@
+import React from "react"
+import Section from "./Section"
+import styled from "styled-components"
+
+const Grid = styled.div`
+  display: grid;
+  grid-column-gap: 20px;
+  grid-row-gap: 20px;
+  grid-template-columns: 1fr 1fr;
+  grid-auto-flow: dense;
+`
+
+const DownloadLink = styled.a`
+  display: flex;
+  padding: 20px;
+  align-items: center;
+  justify-content: space-between;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  line-height: 15px;
+  letter-spacing: 0.14em;
+  text-transform: uppercase;
+  color: var(--sunrise-gray);
+  fill: currentColor;
+  stroke: currentColor;
+  background-color: var(--sunrise-tan);
+  text-decoration: none;
+`
+
+/**
+ * @type {React.ReactNode}
+ */
+const icon = (
+  <svg width="13" height="17" viewBox="0 0 13 17">
+    <rect
+      x="0.619048"
+      y="1.38077"
+      width="11.7619"
+      height="14.2381"
+      rx="1.85714"
+      strokeWidth="1.2381"
+      fill="none"
+    />
+    <rect
+      x="2.4762"
+      y="4.47614"
+      width="8.04762"
+      height="1.2381"
+      stroke="none"
+    />
+    <rect
+      x="2.4762"
+      y="7.57135"
+      width="8.04762"
+      height="1.2381"
+      stroke="none"
+    />
```

Comment:
Gut feeling is this whole icon should live in it's own file.
But personally preference aside, lgtm!

Reply:
Eh, looking at the repo's file structure and how it's most likely to only be used on document.
I 100% get it being there.

Def see it being pulled out in future.
If only for how much space it takes up in Document alone lol.

Reply:
Yeah the code formatter is kind of aggressive here and I'm not sure how to disable it for SVG, where we're never going to actually read the props. I'll leave it here for now.